build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -269,8 +271,10 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
                     Exec(msbuildPath, cmakeOutPath, configParameter);
 
                 // Copy the output out
-                File.Copy(Path.Combine(cmakeOut, "cli", "exe", configuration, "dotnet.exe"), Path.Combine(Dirs.CorehostLatest, "dotnet.exe"), overwrite: true);
-                File.Copy(Path.Combine(cmakeOut, "cli", "exe", configuration, "dotnet.pdb"), Path.Combine(Dirs.CorehostLatest, "dotnet.pdb"), overwrite: true);
+                File.Copy(Path.Combine(cmakeOut, "cli", "exe", "dotnet", configuration, "dotnet.exe"), Path.Combine(Dirs.CorehostLatest, "dotnet.exe"), overwrite: true);
```

Comment:
NIT: My suggestion here would be to refactor this to a loop that works off an array of host names.